intro/scipy/examples/plot_fftpack.py

```diff
@@ -28,7 +28,7 @@
 period = 5.0
 
 time_vec = np.arange(0, 20, time_step)
-sig = np.sin(2 * np.pi / period * time_vec) + 0.5 * np.random.randn(time_vec.size)
+sig = np.sin(2 * np.pi / period * time_vec) + 0.5 * rng.randn(time_vec.size)
```

Comment:
There are many instances where an Generator is available but is unused. I'll create a separate PR for that.

Reply:
```suggestion
sig = np.sin(2 * np.pi / period * time_vec) + 0.5 * rng.normal(size=time_vec.size)
```